### PR TITLE
Feature/sort newslist

### PR DESCRIPTION
**Pull request by Delf-Lee**
### 이런게 변경되었어요. 💍
- Admin에서의 New 리스트를 시간 역순으로 정렬하도록 변경했습니다.

### CODE REVIEW

**Comment by chul-hyun** [comment]
Hey @Delf-Lee,  
Your changes look good to me!

<a href="https:&#x2F;&#x2F;travis-ci.org&#x2F;snack-news&#x2F;Snack-BE&#x2F;builds&#x2F;612415717">View build log</a>

###### TravisBuddy Request Identifier: d0492bc0-07b6-11ea-ae60-a90e33eef11d

**Comment by NESOY** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 5
           

Complexity increasing per file
==============================
- src/main/java/com/snack/news/service/AdminService.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Snack-BE/Snack-BE/pullRequest?prid=4508329&bid=15193214)

File: src/test/java/com/snack/news/repository/NewsRepositoryTest.java [hunk 1]
```diff
@@ -13,17 +13,21 @@
...
+import org.springframework.data.domain.Sort;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.time.LocalDateTime;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.Comparator;
 import java.util.List;
+import java.util.stream.Collectors;
```
Comment (NESOY) [inline src/test/java/com/snack/news/repository/NewsRepositoryTest.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.stream.Collectors'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

Comment (NESOY) [inline src/test/java/com/snack/news/repository/NewsRepositoryTest.java]: Codacy Issue found: [Unused import - java.util.stream.Collectors.](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

File: src/test/java/com/snack/news/repository/NewsRepositoryTest.java [hunk 2]
```diff
@@ -203,4 +207,31 @@
...
+		List<News> expectedListFistNewsPage = newsRepository.findAll()
+				.stream().sorted(Comparator.comparing(News::getId).reversed()).limit(pageSize).collect(toList());
+
+		long lastNewsId1 = expectedListFistNewsPage.get(0).getId();
+		long lastNewsId2 = expectedListFistNewsPage.get(1).getId();
+		long lastNewsId3 = expectedListFistNewsPage.get(2).getId();
+		long lastNewsId4 = expectedListFistNewsPage.get(3).getId();
+		long lastNewsId5 = expectedListFistNewsPage.get(4).getId();
+		
+		Pageable pageableDesc = PageRequest.of(0, pageSize, new Sort(Sort.Direction.DESC, "id"));
```
Comment (NESOY) [inline src/test/java/com/snack/news/repository/NewsRepositoryTest.java]: Codacy Issue found: [Avoid unused local variables such as 'pageableDesc'.](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

File: src/test/java/com/snack/news/service/AdminServiceTest.java [hunk 3]
```diff
@@ -18,7 +19,8 @@
 import java.util.List;
 
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
```
Comment (NESOY) [inline src/test/java/com/snack/news/service/AdminServiceTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

File: src/test/java/com/snack/news/controller/AdminControllerTest.java [hunk 4]
```diff
@@ -16,13 +18,20 @@
...
 import java.util.Collections;
+import java.util.Objects;
 
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.when;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
```
Comment (NESOY) [inline src/test/java/com/snack/news/controller/AdminControllerTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.springframework.test.web.servlet.request.MockMvcRequestBuilders'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

**Comment by NESOY** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 2
           

Complexity increasing per file
==============================
- src/main/java/com/snack/news/service/AdminService.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Snack-BE/Snack-BE/pullRequest?prid=4508329&bid=15193214)

File: src/test/java/com/snack/news/controller/AdminControllerTest.java (hunk 4 above)
Comment (NESOY) [inline src/test/java/com/snack/news/controller/AdminControllerTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.springframework.test.web.servlet.request.MockMvcRequestBuilders'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

File: src/test/java/com/snack/news/service/AdminServiceTest.java (hunk 3 above)
Comment (NESOY) [inline src/test/java/com/snack/news/service/AdminServiceTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

**Comment by chul-hyun** [comment]
Hey @Delf-Lee,  
Your changes look good to me!

<a href="https:&#x2F;&#x2F;travis-ci.org&#x2F;snack-news&#x2F;Snack-BE&#x2F;builds&#x2F;612796625">View build log</a>

###### TravisBuddy Request Identifier: 507b9bf0-0877-11ea-9a27-7119c71b9423

File: src/test/java/com/snack/news/service/AdminServiceTest.java (hunk 3 above)
Comment (NESOY) [inline src/test/java/com/snack/news/service/AdminServiceTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)

**Comment by NESOY** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/com/snack/news/service/AdminService.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Snack-BE/Snack-BE/pullRequest?prid=4508329&bid=15193214)

**Comment by NESOY** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
           

Complexity increasing per file
==============================
- src/main/java/com/snack/news/service/AdminService.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Snack-BE/Snack-BE/pullRequest?prid=4508329&bid=15193214)

**Pull request merged**